### PR TITLE
add TippyFlitsUK to editor

### PR DESCRIPTION
**Pull request by jennijuju**
4 editors approved this addition 
image

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoares
+*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoare @TippyFlitsUK
```
Comment (kaitlin-beegle) [inline CODEOWNERS:1]:
```suggestion
*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoares @TippyFlitsUK
```

I assume this was an accidental typo, but Jorge's handle is still @jsoares with the "s"

**Review by kaitlin-beegle — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (jsoares) [reply CODEOWNERS]: thx @kaitlin-beegle :)

**Review by jsoares — APPROVED** [review]
lgtm but note that GH is complaining about codeowners being invalid as James does not have write access to the repo. Please make sure to grant him appropriate permissions before/as you merge.

**Comment by jennijuju** [comment]
> lgtm but note that GH is complaining about codeowners being invalid as James does not have write access to the repo. Please make sure to grant him appropriate permissions before/as you merge.

added him to GH editor group

**Pull request merged**